lib/console_web/controllers/gateway_controller.ex

```diff
@@ -66,4 +68,25 @@ defmodule ConsoleWeb.GatewayController do
 
     Absinthe.Subscription.publish(ConsoleWeb.Endpoint, gateway, gateway_added: "#{gateway.team.id}/gateway_added")
   end
+
+  defp geocodeLatLng(%{"latitude" => lat, "longitude" => lng}) do
```

Comment:
this might be worth at least moving to the Gateways context, if not to its own module